admin/src/components/Processors/EditProcessor.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, Text, TextArea, Checkbox } from 'react-form';
+
+import Loader from '../core/cards/dashboard/Loader';
+
+export const EditProcessor = props => {
+  const scopes = {
+    'Full Name': 'user.fullName',
+    Email: 'user.email',
+    'Phone Number': 'user.phoneNumber'
+  };
+
+  const renderForm = () => (
+    <React.Fragment>
+      <label htmlFor="name">Name:</label>
+      <Text field="name" id="name" />
```

Comment:
Have you seen how I used our `TextInput` control and validation logic in a container?
Maybe we should consider using that approach – I think it will cut down on the code a little bit and make it consistent across different forms. Let's chat about it after standup.

Reply:
Yep! I saw that, just flagged the PR as WIP, to check with you how we can avoid repeating the part of code related to events on form controls, perhaps you have some ideas after making the user forms. Let discuss this a little bit!